README.md

```diff
@@ -10,6 +10,71 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple classes in the same file, each class should have its own file. If you want to create a private class you can either define a private class inside the same file, or create a nested class inside the file's class.
+
+```swift
+class myClass {
+	// You can create a nested class inside the class...
+	class myNeastedClass {
+	}
+}
+
+// ...or create a private class inside the file.
+private class myPrivateClass {
+}
+```
+
+The following list should be the standard struct/class organization of all your Swift files, in this specific order:
+
+Before the class declaration:
+
+* Enums (as private)
+* Static variables
```

Comment:
This needs to go into the type declaration (inside the class). 

Public Properties
Private Properties
Public Static
Private Static

Init/Deinit
Override
Public functions
Private functions